src/Arcus.Templates.AzureFunctions.ServiceBus.Queue/OrdersAzureServiceBusMessageHandler.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Arcus.Messaging.Abstractions;
+using Arcus.Messaging.Abstractions.ServiceBus;
+using Arcus.Messaging.Abstractions.ServiceBus.MessageHandling;
+using Arcus.Templates.AzureFunctions.ServiceBus.Queue.Model;
+using GuardNet;
+using Microsoft.Extensions.Logging;
+
+namespace Arcus.Templates.AzureFunctions.ServiceBus.Queue
+{
+    /// <summary>
+    /// Represents an <see cref="IAzureServiceBusMessageHandler{TMessage}"/> implementation that processes <see cref="Order"/> messages.
+    /// </summary>
+    public class OrdersAzureServiceBusMessageHandler : IAzureServiceBusMessageHandler<Order>
```

Comment:
Other than the worker template, I chose to use an 'order' in our project template instead of the `EmptyMessage` variant currently in the worker template. Seemed like a better and more understandable way when first opening the project.

Reply:
Agree.  It will be more straightforward to understand I think.